docs/source/configuration/authorization.mdx

```diff
@@ -442,46 +441,38 @@ extend schema
     import: [..., "@policy"])
 ```
 
-It is defined as follows:
+The `@policy` directive is defined as follows:
 
 ```graphql
 scalar federation__Policy
 directive @policy(scopes: [[federation__Policy!]!]!) on OBJECT | FIELD_DEFINITION | INTERFACE | SCALAR | ENUM
 ```
 
-Using the `@policy` directive requires a [Supergraph plugin](../customizations/overview) to evaluate the authorization policies. You can do this with a [Rhai script](../customizations/rhai/), [coprocessor](../customizations/coprocessor), or [native plugin](../customizations/native). Refer to the following Rhai script and coprocessor examples for more information.
-
+Using the `@policy` directive requires a [Supergraph plugin](../customizations/overview) to evaluate the authorization policies. You can do this with a [Rhai script](../customizations/rhai/) or [coprocessor](../customizations/coprocessor). Refer to the following [Rhai script](#usage-with-a-rhai-script) and [coprocessor](#usage-with-a-coprocessor) examples for more information. (Although a [native plugin](../customizations/native) can also evaluate authorization policies, we don't recommend using it.)
 
 #### Combining policies with `AND`/`OR` logic
```

Comment:
Should we combine this with the `@requiresScopes` description of AND/OR? I believe it works the same way, but maybe it's better to keep the definition separate, unsure.